rov/movement/controller.py

```diff
@@ -54,7 +68,7 @@ def get_thruster_values(self):
         """The value of each thruster returned as a list"""
         return self.thrusters.get()
 
-    def set_thruster_values(self, values):
+    def set_thruster_avlues(self, values):
```

Comment:
uhh

Reply:
Yeah this might be why Ryan was unable to change the values 🙃
Referencing error? I called it